docs/site/Context.md

```diff
@@ -152,77 +152,11 @@ returns the ContextValue that was initially bound (we can do other fancy things
 too -- ie. instantiate your classes, etc)
 
 The process of registering a ContextValue into the Context is known as
-_binding_. Sequence-level bindings work the same way.
+[Binding](Binding.md). Sequence-level bindings work the same way.
```

Comment:
I am proposing to make it more explicit that the page Binding provides more information: preserve the current text and add a new sentence pointing readers to Binding page.

For example:

```markdown
The process of registering a ContextValue into the Context is known as
_binding_. Sequence-level bindings work the same way.

You can learn more about the binding process in [Binding](./Binding.md).
```

BTW, there is no `ContextValue` type anymore, we use `BoundValue` type instead. It would be great to update our docs to use the new name. I am ok to leave that out of scope of this pull request if you prefer.